src/lib/error.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	/**
+	 * @param  {[type]}
+	 * @return {Boolean}
+	 * returns a predicate, that checkes if given error is of the given type. 
+	 * Predicate can be used for passing to bluebird .catch(errorPredicate, errorHandler)
+	 */
+	isOfType : function(type){
```

Comment:
it was used. Then we decided not to do anything special for database errors so this is redundant as of now. Should I remove it ?